cdci_data_analysis/analysis/parameters.py

```diff
@@ -475,10 +507,20 @@ class Name(String):
 
 class Float(Parameter):
     owl_uris = ["http://www.w3.org/2001/XMLSchema#float"]
-    def __init__(self, value=None, units=None, name=None, allowed_units=None, default_units=None, check_value=None):
+    def __init__(self, 
+                 value=None, 
+                 units=None, 
+                 name=None, 
+                 allowed_units=None, 
+                 default_units=None, 
+                 check_value=None, 
+                 min_value= None,
+                 max_value = None):
 
         if check_value is None:
             check_value = self.check_float_value
+        
+        self.check_bounds = basic_numeric_check_bounds
```

Comment:
This somehow mimics what would be more naturally done by an additional intermediate "BoundedParameter" (NumericParameter?)

Reply:
I don't like "BoundedParameter" as bounds are optional. NumericParameter is ok, it could be implemented like this, if you think it's more natural. 

About general comparison operation, we should be careful with this. Python strings, e.g., have comparison defined, but it's meaningless in most cases

Reply:
more like "boundable" then. But Numeric is a useful specialization of it which we can use.